src/ParallelAlgorithms/Events/ObserveNorms.hpp

```diff
@@ -350,11 +344,7 @@ operator()(const typename ObservationValueTag::type& observation_value,
                                                 Frame::Inertial>>(box));
   const size_t number_of_points = mesh.number_of_grid_points();
   const double local_volume = [&mesh, &det_jacobian]() {
-    if (mesh.basis(0) == Spectral::Basis::FiniteDifference) {
```

Comment:
Ah, the issue is that in this case is that the integrand of whatever the code is trying to take a volume integral of, i.e. the error terms, can't be done on the FD grid because the compute tag that computes the integrand isn't aware that it needs to know whether the grid is FD or DG. However, as far as I can tell the TOV star is using the right `error_compute` (see line 343 of `src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp`). Can you reproduce the error locally and get into a debugger to figure out a backtrace?